src/engine-actions/get-status.js

```diff
@@ -0,0 +1,76 @@
+const grpc = require('grpc')
+
+const {
+  getInfo,
+  genSeed
+} = require('../lnd-actions')
+
+/**
+ * @constant
+ * @type {String}
+ * @default
+ */
+const WALLET_EXISTS_ERROR_MESSAGE = 'wallet already exists'
```

Comment:
I would add some description about what this is, including a link to the LND docs or code where it is defined.